helpers.py

```diff
@@ -29,3 +33,15 @@ def wait_ten_seconds_and_close(browser):
     if browser:
         time.sleep(10)
         browser.quit()
+
+
+def check_if_user_is_authorized(browser):
+    """Check if user is authorized"""
+    if browser:
+        try:
+            popup_is_closed = WebDriverWait(browser, 5).until(
+                ec.invisibility_of_element_located((By.ID, "login_form"))
+            )
+            return popup_is_closed
```

Comment:
Объясни, пожалуйста, почему так лучше
```suggestion
            return True
```

Reply:
Лучше оставить true для читаемости. Иначе нужно перечитать что из себя представляет popup_is_closed, чтобы понять что возвращает функция.

Reply:
А что она возвращает?

Reply:
true/false возвращает

сразу true надо было писать) так прямо точно понятнее

Reply:
[Документация](https://www.selenium.dev/selenium/docs/api/py/webdriver_support/selenium.webdriver.support.expected_conditions.html#selenium.webdriver.support.expected_conditions.invisibility_of_element_located) 😉